plugins/coverbrowser.koplugin/main.lua

```diff
@@ -469,7 +469,7 @@ function CoverBrowser:refreshFileManagerInstance(cleanup, post_init)
         if cleanup then -- clean instance properties we may have set
             if fc.onFileHold_orig then
                 -- remove our onFileHold that extended file_dialog with new buttons
-                fc.onFileHold = fc.onFileHold_orig
+                fc.showFileDialog = fc.onFileHold_orig
                 fc.onFileHold_orig = nil
                 fc.onFileHold_ours = nil
```

Comment:
Shouldn't all these `onFileHold_orig/ours` also be renamed to `showFileDialog_orig/ours` ?
Are we backuping/replacing/restoring only` :showFileDialog` ? or really `:onFileHold` ?
(Unless you're unconfortable tweaking that ugly twisted hijacking coverbrowser code - which I can understand :)

Reply:
https://github.com/koreader/koreader/blob/73ca1e219373f96275928fe6136787fe27233f25/frontend/apps/filemanager/filemanager.lua#L184-L190
`:showFileDialog` only is backuped.
As it is the main part of `onFileHold`, and to avoid massive renaming and to keep some consistency with the code of replacing/backuping of History/Colections buttons, I've kept `onFileHold_orig` and `onFileHold_ours`.

Reply:
Well, second thoughts: this makes this section of coverbrowser a bit awkward/ugly/lying :)
Even just adding a comment "we used to replace onFileHold, which has become a small wrapper; we replace now the inner method" would still leave it awkward.
So, either do the massive renaming (because, no, this consistency with false identity is not the consistency you are known for :).
Or also add the `if file_manager.select_mode then file_manager:tapPlus() ; return end` (that you added in the original onFileHold) in the Coverbrowser method that replaces it (dunno if that's possible, if you have access to `file_manager.select_mode` in there...).